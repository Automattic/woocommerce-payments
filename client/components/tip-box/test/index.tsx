/**
 * External dependencies
 */
import React from 'react';
import { render } from '@testing-library/react';

/**
 * Internal dependencies
 */
import TipBox from '../';

describe( 'TipBox Component', () => {
	it( 'renders a purple tip', () => {
		const { container } = render(
			<TipBox color="purple">Lorem ipsum</TipBox>
		);
		expect( container ).toMatchSnapshot();
	} );
} );
