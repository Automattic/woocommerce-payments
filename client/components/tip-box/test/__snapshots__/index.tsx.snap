// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TipBox Component renders a purple tip 1`] = `
<div>
  <div
    class="wcpay-component-tip-box purple"
  >
    <svg
      height="24"
      viewBox="-4 -4 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M8 1.33a4.67 4.67 0 0 0-2.67 8.5v1.5c0 .37.3.67.67.67h4c.37 0 .67-.3.67-.67v-1.5A4.66 4.66 0 0 0 8 1.33ZM6 14c0 .37.3.67.67.67h2.66c.37 0 .67-.3.67-.67v-.67H6V14Zm3.33-4.87.57-.4a3.33 3.33 0 1 0-3.8 0l.57.4v1.54h2.66V9.13Z"
        fill-rule="evenodd"
      />
    </svg>
    <div
      class="wcpay-component-tip-box__content"
    >
      Lorem ipsum
    </div>
  </div>
</div>
`;
